src/test/java/io/personium/test/unit/core/rs/odata/DecideOutputFormatTest.java

```diff
@@ -112,6 +136,30 @@
         assertEquals(MediaType.APPLICATION_ATOM_XML_TYPE, type);
     }
 
+    /**
+     * acceptの空白とセミコロン以降を無視すること.
+     */
+    @Test
+    public final void acceptの空白とセミコロン以降を無視すること() {
+        ODataEntityResource odataEntityResource = new ODataEntityResource();
+
+        MediaType type = odataEntityResource.decideOutputFormat(
+                "application/xml ;q=0.9,*/* ;q=0.8", null);
+        assertEquals(MediaType.APPLICATION_ATOM_XML_TYPE, type);
+    }
+
+    /**
+     * acceptのタブとセミコロン以降を無視すること.
+     */
+    @Test
+    public final void acceptのタブとセミコロン以降を無視すること() {
+        ODataEntityResource odataEntityResource = new ODataEntityResource();
+
+        MediaType type = odataEntityResource.decideOutputFormat(
+                "application/xml\t;q=0.9,*/\t;q=0.8", null);
```

Comment:
Just a nitpicking: `*/\t` → `*/*\t`

Reply:
Thank you for you review. I modified it. After automatic testing, I will merge this PR.